app/admin/member.rb

```diff
@@ -1,4 +1,12 @@
 ActiveAdmin.register Member do
   permit_params :email, :country, :first_name, :last_name, :city, :postal, :title, :address1, :address2, :actionkit_user_id
   actions :all, except: [:destroy]
+
+  sidebar 'Previous Versions', only: :show do
```

Comment:
isn't there a sort of partial system that you could use to DRY this code up a bit? it's a lot of copy and paste of these 7 lines.